loopin.go

```diff
@@ -180,7 +187,12 @@ func newLoopInSwap(globalCtx context.Context, cfg *swapConfig,
 		return nil, fmt.Errorf("cannot store swap: %v", err)
 	}
 
-	return swap, nil
+	swap.log.Infof("Server message: %v", swapResp.serverMessage)
```

Comment:
Perhaps if `!=""`, otherwise we log `Server message: `

Reply:
added